mindmap/package.json

```diff
@@ -11,7 +11,8 @@
     "lint-fix": "eslint . --ext .js,.ts,.cjs,.json,.tsx --fix",
     "lint-styles": "stylelint src/index.css",
     "lint-css": "ilcd",
-    "knip": "knip"
+    "knip": "knip",
+    "test": "node --test"
```

Comment:
feel free to use mocha, c8 and chai like in all other parts :)

Reply:
Understood @Idrinth ill make the requested changes :) .

Reply:
The benefit there is mostly coverage information, so we can actually see how well or badly the project is tested